nature_importer.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+import argparse
+import csv
+import os
+import importer_utils as utils
+import wikidataStuff.wdqsLookup as lookup
+from NatureArea import NatureArea
+from PreviewTable import PreviewTable
+from Uploader import Uploader
+
+DATA_DIRECTORY = "data"
+reserves_file = "NR_polygon.csv"
+nationalparks_file = "NP_polygon.csv"
+edit_summary = "test"
+
+
+def get_data_from_csv_file(filename):
+    """Load data from csv file into a list."""
+    with open(filename, "r") as f_obj:
+        reader = csv.DictReader(f_obj, delimiter=',')
+        csv_data = list(reader)
+    return csv_data
+
+
+def remove_duplicate_entries(nature_dataset):
+    """
+    Remove duplicate entries from Nature Reserve file.
+
+    In cases where a reserve occurs twice, it's because
+    of different values of "BESLSTATUS". Whenever that happens,
+    one of them is always "gällande". This is the one we keep,
+    and the other one is removed.
+    """
+    results = []
+    unique_ids = []
+    for row in nature_dataset:
+        n_id = row["NVRID"]
+        if n_id in unique_ids:
+            new_status = row["BESLSTATUS"]
+            status_in_results = [x["BESLSTATUS"]
+                                 for x in results if x["NVRID"] == n_id]
+            if status_in_results[0] == "Gällande":
+                pass
+            elif new_status == "Gällande":
+                utils.remove_dic_from_list_by_value(results, "NVRID", n_id)
+                results.append(row)
+        else:
+            unique_ids.append(n_id)
+            results.append(row)
+    return results
+
+
+def load_nature_area_file(which_one):
+    """
+    Load source file with nature area data.
+
+    :param which_one: nr for reserves or np for parks.
+    """
+    if which_one == "nr":
+        filepath = os.path.join(DATA_DIRECTORY, reserves_file)
+    elif which_one == "np":
+        filepath = os.path.join(DATA_DIRECTORY, nationalparks_file)
+    print("Loading dataset: {}".format(filepath))
+    dataset = get_data_from_csv_file(filepath)
+    no_duplicates = remove_duplicate_entries(dataset)
+    return no_duplicates
+
+
+def get_wd_items_using_prop(prop):
```

Comment:
Other than the striping of the "Q" from the ids does this differ form `helpers.fill_cache_wdqs()` in WikidataStuff?